services/frontend-service/src/ui/NavigationBar/NavigationBar.tsx

```diff
@@ -21,7 +21,7 @@ import { ReactComponent as Locks } from '../../images/Locks.svg';
 import { NavList, NavListItem } from '../components/navigation';
 
 export const NavigationBar: React.FC = () => (
-    <aside className="mdc-drawer">
+    <aside className="mdc-drawer" id={'NavigationBar'}>
```

Comment:
Do you actually need the `id` here? We should avoid assigning ids to elements

Reply:
Not necessarily, since it was used for testing. Will change it!